canvas/models/models.py

```diff
@@ -17,6 +17,7 @@ class CanvasCourse(models.Model):
     course_id = models.IntegerField()
     token = models.CharField(max_length=500)
     instructor = models.ForeignKey(MyUser, on_delete=models.SET_NULL, null=True, blank=True)
+    description = models.CharField(max_length=2000, null=True, blank=True)
```

Comment:
max_length 2000 is a lot I think. Let's keep it to 500 unless you think we need more, in that case, let's have a chat.

Reply:
No, I think 500 should be fine, I just was not certain how high to set it.